static/src/js/components/MetadataForm/MetadataForm.scss

```diff
@@ -14,15 +14,13 @@
 .rjsf {
   .custom-array-field-template__description,
   .field-description {
-    margin-top: 1rem;
-    margin-bottom: 2rem;
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
     color: $gray-800;
     font-size: 0.925rem;
   }
 
   .row.row-children {
-    margin-bottom: 3rem;
```

Comment:
I see why you got rid of this, but I think it's necessary to at least keep some padding (perhaps a 1rem?) As it removes all padding from certain areas (ie. Collection >> Data Identification >> Collection Data Type)

Reply:
No, it has some bad side affects in a lot of places.   Try adding it back in and you'll see what I mean.  I'm not saying the extra padding shouldn't be solved, just saying this way of doing it I things causes more issues than it helps.